src/urlMatcherFactory.js

```diff
@@ -623,7 +623,7 @@ function $UrlMatcherFactory() {
     any: { // does not encode/decode
       encode: angular.identity,
       decode: angular.identity,
-      is: angular.identity,
+      is: function(){ return true; },
```

Comment:
@eddiemonge The reason for this is that any was always returning whatever you passed in.

`any.is(false)` would always return `false`.  Since everything is type `any` it should always return true.